### PR TITLE
chore: UI Improvements of documentation

### PR DESCRIPTION
**Pull request by alyssapatricia**
# Detailed Description

### Summary
 
> This PR creates an improved documentation: [Improved Documentation](https://alyssapatricia.github.io/ui/)

 
- Changed the theme from Jekyll to Sphinx
- Organized the Topic
- Added a Table of Contents panel
- Added a search engine
- Proofread the existing contents
- Improved overall UI

 
### Motivation & Context

- To improve the documentation of the UPRI EarthquakeHub Project

### Dependencies

- [sphinx](https://www.sphinx-doc.org/en/master/) is the documentation generator
- [sphinx_rtd_theme](https://sphinx-rtd-theme.readthedocs.io/) is the theme of the documentation. 
- [myst_parser](https://pypi.org/project/myst-parser/) is used to parse the .md files (the format which the content files were written) to make it readable by the sphinx environtment
- [sphinxcontrib-openapi](https://github.com/sphinx-contrib/openapi) to deploy API docs

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by chrscbs** [comment]
Hello, Pat. I have successfully cloned your work your repository to mine. Now, I am deploying it to github pages but I have encountered some error in workflow. I have followed your instructions in the readme file. Providing here the screenshots of the error I encountered, please advice on what to do. Thanks!

image

image
image

File: README.md [hunk 1]
```diff
@@ -1,32 +1,89 @@
...
+```
+
+4. **Open the Documentation:** Depending on the format, you can access the documentation using a web browser for HTML-based documentation, or a text editor for Markdown, RST or other text-based formats.
+
+
+
+## Deploying on GitHub Pages
+To deploy your documentation on GitHub Pages, follow these steps:
+
+1. Configure GitHub Pages: In your Forked GitHub repository, go to the "Settings" tab and scroll down to the "GitHub Pages" section. Set the source branch to "gh-pages" and the "docs" folder. Save the changes.
```
Comment (chrscbs) [inline README.md]: Okay, it now works by changing the required directory to use in building the github-pages from /docs to /(root).

Please change the instruction under README. Thanks

**Review by chrscbs — CHANGES_REQUESTED** [review]

**Comment by chrscbs** [comment]
image

Whenever I click this link, this doesn't redirect to the right link. Please fix.

**Comment by chrscbs** [comment]
Comments:

1. Whenever I click this link, this doesn't redirect me to the right link. Please fix this. 
image
2. Please fix this as well.
image
3. And also this one.
image
4. Can we also do opening of clickable links in a new tab whenever a link is clicked, instead of directly redirecting through the same tab in browser. This is for all the clickable links within the documentation.
5. Page content is not occupying the whole screen. Leaving a large portion of the page blank. 
image

**Comment by chrscbs** [comment]
Laki ng improvement ng UI. Grabeeee! Lupet! 

Last minimal request for change; after this approve na 'tong PR for me. I also checked yung api-docs, grabe din improvement. Okay din contents. Kumpleto lahat ng endpoints.
1. This is the last request for change: 
Pakichange yung info na ito under this page: https://alyssapatricia.github.io/ui/sending-data-to-ehub-network.html
image

**Comment by prokorpio** [comment]
These need to be fixed:
1. This probably looks better in white
image
2. Improve "make understanding more significant" 
image
3. In FDSNWS tutorial, "fingerprint" is a more precise analogy than "manual"
image
4. The sender-frontend section is missing.
image
image


Other than those mentioned above, this is a huge improvement for the documentation :) Plus points for the following:
1. Most of the explanation in the FDSNWS tutorial are well thought-out to fit our target audience.
2. Examples and additional links in the FDSNWS tutorial are also very effective add-ons.
3. The new figures for the topic discussions are also effective in communicating the concepts.

I will merge this now but will also create an [issue](https://github.com/UPRI-earthquake/upri-earthquake.github.io/issues/2) to address the above. Thank you.

**Pull request merged**